### PR TITLE
Add Actual Budget to Money, Budget, Management category.

### PR DESCRIPTION
**Pull request by d-kholin**
Added Personal Finance Manager YNAB clone.

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x ] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [ x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x ] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x ] Additions are inserted preserving alphabetical order.
- [ ] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [axwesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [ x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [ x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x ] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [ x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [ x] Any software project you are adding to the list is actively maintained.
- [x ] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1336,6 +1336,7 @@ Software that does not fit in another section.
 
 _See also: [awesome-sysadmin/IT Asset Management](https://github.com/awesome-foss/awesome-sysadmin#it-asset-management)_
 
+- [Actual Budget](https://actualbudget.com) - This is Actual, a local-first personal finance tool. It is now 100% free and open-source. ([Demo](https://app.actualbudget.com/budget) [Source Code](https://github.com/actualbudget/actual-server)) `MIT` `JavaScript`
```
Comment (KieranRobson) [inline README.md:1339]: > yarn install
yarn start

> https://github.com/actualbudget/actual-server#running-via-docker

- Please change the Language tag to `Nodejs/Docker`

Comment (KieranRobson) [inline README.md:1339]: - Please add the missing comma after the demo link

`([Demo](https://app.actualbudget.com/budget), [Source Code](https://github.com/actualbudget/actual-server))`

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Couple changes to be made!

File: README.md (hunk 1 above)
Comment (d-kholin) [reply README.md]: Fixed as per suggested edits, thanks.

File: README.md [hunk 2]
```diff
@@ -1336,6 +1336,7 @@ Software that does not fit in another section.
 
 _See also: [awesome-sysadmin/IT Asset Management](https://github.com/awesome-foss/awesome-sysadmin#it-asset-management)_
 
+- [Actual Budget](https://actualbudget.com) - This is Actual, a local-first personal finance tool. It is now 100% free and open-source. ([Demo](https://app.actualbudget.com/budget), [Source Code](https://github.com/actualbudget/actual-server)) `MIT` `Nodejs/Docker`
```
Comment (nodiscc) [inline README.md:1339]: Please change the description to something more meaningful, e.g. the one found on the [website](https://actualbudget.com/)

> Actual is a super fast privacy-focused app for managing your finances.

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by nodiscc** [comment]
No reply for > a month, closing.

**Pull request closed without merging**